src/internal/file_operations_compress.go

```
@@ -2,44 +2,54 @@ package internal
+func zipSources(sources []string, target string) error {
 	id := shortuuid.New()
 	prog := progress.New()
 	prog.PercentageStyle = common.FooterStyle
+	var err error = nil
 
-	totalFiles, err := countFiles(source)
-
-	if err != nil {
-		slog.Error("Error while zip file count files ", "error", err)
+	for _, src := range sources {
+		if _, err := os.Stat(src); os.IsNotExist(err) {
```

golangci-lint (1.64.8)

shadow: declaration of "err" shadows declaration at line 22

(govet)